src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3937,7 +3937,8 @@ internal void UpdateTypes(ExecutionContext context, bool updateOnly)
 
             if (errors.Count > 0)
             {
-                var allErrors = new StringBuilder('\n');
+                // Bug: don't use '\n' ('char' convert implicitly to 'ushort' and 'int' and initialize 'StringBuilder' size)
```

Comment:
Good catch @iSazonov 👍 
Can you change `Bug:` to `Note:` please? `Bug:` usually indicates there is a potential bug here that needs to be revisited.